widgetssdk/src/main/java/com/glia/widgets/GliaWidgets.java

```diff
@@ -39,7 +39,12 @@ public class GliaWidgets {
     /**
      * Use with {@link android.os.Bundle} to pass in a {@link UiTheme} as a navigation argument when
      * navigating to {@link com.glia.widgets.chat.ChatActivity}
+     *
+     * @deprecated While UiTheme can still be used for UI customization,
+     * we strongly encourage adopting remote configurations {@link GliaWidgetsConfig.Builder#setUiJsonRemoteConfig(String)}.
+     * Remote configurations approach is more versatile and better suited for future development.
```

Comment:
Please update the same sentence everywhere.
(Seems that this is again one of such cases (previously discussed [here](https://github.com/salemove/android-sdk/pull/730#discussion_r1660855313)) that the generator does not render for the [SDK docs](http://android-sdk-docs.glia.com.s3-website-us-east-1.amazonaws.com/widgets/javadoc/com/glia/widgets/GliaWidgets.html).)
```suggestion
     * The remote configurations approach is more versatile and better suited for future development.
```

Reply:
We can migrate to an `HTML` type of documentation, that is more Kotlin-friendly and has a way better UI and user experience.

![image](https://github.com/salemove/android-sdk-widgets/assets/19994238/982b9d44-0644-480b-be79-6902f63c90f5)
![image](https://github.com/salemove/android-sdk-widgets/assets/19994238/d5e1349a-e93f-4ee8-b8d5-6441d85f0204)